VizbeeManager.js

```diff
@@ -113,6 +136,28 @@ class VizbeeManager {
     removeAllListeners(eventName) {
         VizbeeNativeEmitter.removeAllListeners(eventName)
     }
+
+    //------------------
+    // Sign in Helpers
+    //------------------
+
+    registerSignInListeners() {
```

Comment:
If these methods are helpers/private, can we prefix them with the underscore `_registerSignInListeners`?

Reply:
Prefixed the helpers with _ since they are private